src/modules/views/partials/footer.handlebars

```diff
@@ -0,0 +1,57 @@
+{{!-- <div class="footer">
+    <table>
```

Comment:
if not require can we remove this commented code.

Reply:
Resovled